hack/build-cross.sh

```diff
@@ -20,6 +25,8 @@ OS_BUILD_PLATFORMS=("${platforms[@]}")
 host_platform=$(os::build::host_platform)
 if [[ $host_platform == "linux/ppc64le" ]]; then
   OS_GOFLAGS_LINUX_PPC64LE="-tags=gssapi" os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+elif [[ $host_platform == "linux/arm64" ]]; then
```

Comment:
Oh, whoops, I mis-read this slightly. We can just do this:

```sh
local platform_goflags_envvar; platform_goflags_envvar=OS_GOFLAGS_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
declare "${platform_goflags_envvar}=-tags=gssapi"
os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
```

If we wanted to support `zsh` or something we could use `typeset` but we are fine with `bash`.